task_templates/transforms/python3_image_transform/custom.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union, Any
+import pandas as pd
+
+from img_utils import (
+    b64_to_img,
+    img_to_b64,
+    img_to_grayscale,
+)
+
+
+def _process_image(raw_data: Union[str, bytes]) -> bytes:
+    img = b64_to_img(raw_data)
+    img = img_to_grayscale(img)
```

Comment:
please use different names, ex:
```suggestion
    initial_img = b64_to_img(raw_data)
    grayscale_img = img_to_grayscale(initial_img)
```